java/arcs/core/host/ArcHost.kt

```diff
@@ -11,7 +11,7 @@
 
 package arcs.core.host
 
-import arcs.core.data.ParticleSpec
+import arcs.core.data.Plan.Particle
```

Comment:
Totally optional, but I would have imported `Plan`, and referred to the Particle class as `Plan.Particle` to make it very clear that it's the Plan particle rather than the real particle. Here and throughout, but it's really not that important in most files (here it probably is more important though)

Reply:
Yes, that's obviously what should have happened. Done everywhere.